docs/gsod.md

```diff
@@ -72,3 +72,76 @@ ELRS has set its next major release goal before November to include release note
 ## Additional information
 
 The ELRS development team includes a range of expertise, from software professionals, academics, researchers, scientists, and university students.  And we love nothing more than nerding out on all things FPV in our free time :smiley:
+
+## Case study
+
+[**ExpressLRS**](https://www.expresslrs.org)
+
+Organization Description:
+
+ExpressLRS (ELRS) is an open source project focusing on developing a radio control (RC) link. The link is primarily designed for first-person view (FPV) crafts (e.g Multicopters, Planes). ELRS aims to provide the best, completely open source, high refresh radio control link, minimising latency while maximising range. A vast range of hardware in both 900 Mhz and 2.4 GHz frequencies is available.
+
+FPV is a community built on top of open source projects such as [flight controller firmware Betaflight](https://github.com/betaflight) and [RC handset firmware EdgeTX](https://github.com/EdgeTX).  ELRS fills a void for an open source and high performance RC link with an active development team of about 20 volunteers.
+
+[github.com/ExpressLRS](https://github.com/ExpressLRS/ExpressLRS) was created nearly 4 years ago and in the past 12 months has experienced significant growth.  Not only in code contributions, but also users and commercial manufacturers releasing ELRS compatible hardware.
+
+**Problem Statement / Proposal Abstract**
+
+During early ELRS development users were required to Do It Yourself (DIY) and build their own hardware. This tended to attract tech savvy users. However, in the past 12 months a number of commercial manufacturers have released ELRS compatible hardware. A much wider user group now exists of varying knowledge and initial setup of ELRS can be troublesome. More so for users who are also new to FPV.
+
+A review of the existing user documentation by a technical writer, who has not previously been exposed to FPV, will bring a fresh pair of eyes to the project and help identify knowledge gaps.
+
+**Project Description**
+
+**Creating the proposal**
+
+The proposal idea was pretty easy for us… as we like to keep it simple.  GSoD is about documents, and hiring a Technical Writer, so we rubbed our 2 brain cells together and decided we needed help reviewing a writing docs.  The exploding ELRS users base meant we can not continue to provide 1 on 1 support to new users, unlike in the beginning of the project.  The demand for more and better documents is and will be ongoing during the ELRS growth period.  However, the community appears to have reached a critical mass and the user base is great at supporting new users.
```

Comment:
Is this supposed to be "decided we needed help reviewing **and** writing docs"?

Reply:
rewriting, making a bit more professional